adapters/rack/lib/opentelemetry/adapters/rack/util/queue_time.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Rack
+      module Util
+        # QueueTime simply...
+        module QueueTime
+          REQUEST_START = 'HTTP_X_REQUEST_START'.freeze
+          QUEUE_START = 'HTTP_X_QUEUE_START'.freeze
+          MINIMUM_ACCEPTABLE_TIME_VALUE = 1_000_000_000
+
+          module_function
+
+          def get_request_start(env, now = Time.now.utc)
```

Comment:
Given the 3 early returns and that `now` is not used until the end of the method, it might be better to default to `now = nil` and defer the `Time.now.utc` until `now` is needed.

Reply:
I've updated as suggested, Thanks!  :heavy_check_mark: